packages/peregrine/lib/talons/CategoryTree/__tests__/useCategoryTree.spec.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import { act } from 'react-test-renderer';
-import { runQuery, queryResult, useLazyQuery } from '@apollo/react-hooks';
+import { runQuery, queryResult, useLazyQuery } from '@apollo/client';
```

Comment:
Not that you added it, but `queryResult` isn't a valid import; even if it were, it's not used that way in this file, so let's remove it.

```suggestion
import { runQuery, useLazyQuery } from '@apollo/client';
```